src/core/server/index.ts

```diff
@@ -396,14 +402,7 @@ export type {
   SavedObjectsImporter,
   SavedObjectsImportError,
 } from './saved_objects';
-export {
-  SavedObjectsClient,
-  SavedObjectsErrorHelpers,
-  SavedObjectsSerializer,
```

Comment:
Last external usage of `SavedObjectsSerializer` have been removed and replaced by the newly introduced mock. It's no longer publicly exposed.